Flow.Launcher.Plugin/SharedCommands/SearchWeb.cs

```diff
@@ -93,7 +93,15 @@ public static void OpenInBrowserTab(this string url, string browserPath = "", bo
                 if (!string.IsNullOrEmpty(browserPath))
                 {
                     psi.FileName = browserPath;
-                    psi.Arguments = (inPrivate ? $"{privateArg} " : "") + url;
+                    if (inPrivate)
+                    {
+                        psi.Arguments = (inPrivate ? $"{privateArg} " : "") + url;    
+                    }
+                    else
+                    {
+                        psi.Arguments = (!String.IsNullOrEmpty(profileArg) ? $"--profile-directory=\"{profileArg}\" " : "") + url;
```

Comment:
Will firefox works for this? Though I don't even believe firefox has a good profile system (it has a nice container feature though but that doesn't require this)

Reply:
no, I think you're right